src/components/VideoPlayer/VideoPlayer.component.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import Divider from '@material-ui/core/Divider';
+import PropTypes from 'prop-types';
+
+import {
+  StyledVideoContainer,
+  StyledVideo,
+  StyledVideoTitle,
+  StyledVideoDescription,
+  StyledVideoTitleContainer,
+  StyledVideoLike,
+} from './styled';
+
+function VideoPlayer({ videoId, title, description }) {
+  const videRole = 'youtube-video';
+  return (
+    <StyledVideoContainer>
+      <StyledVideo
+        role={videRole}
+        title={title}
+        id="ytplayer"
+        type="text/html"
+        src={`https://www.youtube.com/embed/${videoId}?autoplay=0`}
+        frameBorder="0"
+      />
+      <StyledVideoTitleContainer>
+        <StyledVideoTitle>{title}</StyledVideoTitle>
+        <StyledVideoLike>Me gusta</StyledVideoLike>
+      </StyledVideoTitleContainer>
+      <Divider />
+      <StyledVideoDescription>{description}</StyledVideoDescription>
+    </StyledVideoContainer>
+  );
+}
+
+VideoPlayer.propTypes = {
+  videoId: PropTypes.string,
```

Comment:
I'd suggest to put this as a required prop as your component won't display properly without this. Consider this to other components. A way of knowing if it is required or not is by asking yourself if your component would properly render without that prop.

```suggestion
  videoId: PropTypes.string.isRequired,
```